### DISCUSSION TITLE
Utilization Model and VM Utilization

### DISCUSSION THREAD

**Discussion by HilmyThoriq** [manoelcampos/cloudsim-plus#445] [Q&A] [answered]
Greetings,

I'am doing my research on load balancing algortihm using cloudsim plus and i have several questions.

1. Utilization Model in Cloudsim Plus some mode, which one is the best for research using only utilization stochastic or use trace file such as planetlab? 
2. I want to know each VM Utilization after my simulation done, is there any example in Cloudsim Plus? 

Thanks

**Comment by manoelcampos** [comment, marked as answer]
The kind of utilization model you should use depends on your requirements and the kind of scenario you want to simulate. Using trace files enables you to create more realistic simulation scenarios.

Check the list of exclusive features at the home page. The example link is there.
